build.sbt

```diff
@@ -106,9 +107,12 @@ lazy val core = crossProject(JSPlatform, JVMPlatform)
       "org.typelevel" %%% "cats-laws" % catsVersion % Test,
       "org.typelevel" %%% "cats-effect" % catsEffectVersion % Test,
       "org.typelevel" %%% "discipline-munit" % disciplineMUnitVersion % Test,
-      "org.typelevel" %%% "munit-cats-effect-3" % munitCEVersion % Test
+      "org.typelevel" %%% "munit-cats-effect-3" % munitCEVersion % Test,
     )
   )
+  .jvmSettings(
+    libraryDependencies +="org.bouncycastle" % "bcpkix-jdk15to18" % bouncyVersion % Test
```

Comment:
Is this dependency required to implement these algorithms? Is it possible to implement these algorithms in Node.js and browsers?

Reply:
I don't think so. 
I am using examples from the [Signing HTTP Messages](https://httpwg.org/http-extensions/draft-ietf-httpbis-message-signatures.html) spec referenced in the test suite, and last time I tried I felt like I had to reach out to bouncy-castle. As I am building this for the [an implementation of that spec](https://github.com/bblfish/httpSig), I want to make sure it works. If we can find a way to drop that, I'd love to know.

Reply:
I'm sorry, I wasn't paying close attention when I first looked. I see now that it is a test-only dependency, which is less problematic. But, any tests we write using it we won't be able to re-use on JavaScript which is unfortunate.